core/templates/pages/contributor-dashboard-page/modal-templates/translation-modal.controller.ts

```diff
@@ -68,6 +70,87 @@ angular.module('oppia').controller('TranslationModalController', [
+        ' as in the original English text.',
+        'Preserve punctuation and bolding. If the original content has' +
+        ' bold text, make sure it is bold in Hindi as well. If there' +
+        ' are bullet points, double quotes, etc., make sure that the' +
+        ' translated content also has bullet points and double quotes.',
+        'If the original card has “components” (such as pictures,' +
+        ' links, and equations), these need to be added to the' +
+        ' translated content. You can use the “Copy tool” for this' +
+        ' -- click on the Copy tool and then click on the component you want' +
+        ' to carry over. Also, double-click on the image and translate the ' +
+        ' alt text (and caption, if any).'
+      ],
+      bn:
+      [
+        'Use simple Bangla words that are used in daily communication.' +
+        ' Note that common English words (pencil, etc.) can be written as' +
+        ' transliterations (e.g পেন্সিল ).',
+        'Use proper punctuation.',
+        'Full stop = |',
+        'Use the same voice (active or passive)' +
+        ' as in the original English text.',
+        'Preserve punctuation and bolding. If the original' +
+        ' content has bold text, make sure it is bold in' +
+        ' Bangla as well. If there are bullet points, double quotes,' +
+        ' etc., make sure that the translated content also has bullet' +
+        ' points and double quotes.'
+      ],
+      ar:
+      [
+        'In Oppia, we prefer to use simple words that can be easily' +
+        ' understood by children. For example, we use “تابع قائلًا”' +
+        ' instead of “أردف قائلًا”. Furthermore, the English words that' +
+        ' are used in the Arab society regularly can be translated as' +
+        ' follows; Arabic word (The regularly used English word). For' +
+        ' example, we can translate the word cupcakes this way;' +
+        ' كعك القوالب الصغيرة (cupcakes). ',
+        'Use respectful ways and formal prefixes to address people.' +
+        ' For example, use “سيدي” and “سيدتي”. ',
+        'If the name has a meaning in Arabic, or in English, such' +
+        ' as Baker or Crumb, always use words that indicate that they' +
+        ' are names before writing the name itself. For example,' +
+        ' you can use one of the following words depending on the' +
+        ' context; “السيد، السيدة، العم، الجد، الجدة، الآنسة.”',
+        'Use the same voice (active or passive) as in the' +
+        ' original English Text',
+        'Preserve punctuation and bolding. If the original' +
+        ' content has bold text, make sure it is bold in Arabic' +
+        ' as well. If there are bullet points, double quotes, etc.,' +
+        ' make sure that the translated content also has' +
+        ' bullet points and double quotes.',
+        'Use the hyperlinks to different cards as shown in' +
+        ' the original English Text.',
+      ]
+    };
```

Comment:
Can we move this dict to assets/constants.ts?

Reply:
done